cmd/telegraf/telegraf.go

```diff
@@ -149,11 +152,20 @@ func (t *Telegraf) reloadLoop() error {
 			for _, fConfig := range t.configFiles {
 				if _, err := os.Stat(fConfig); err == nil {
 					go t.watchLocalConfig(signals, fConfig)
-				} else {
-					log.Printf("W! Cannot watch config %s: %s", fConfig, err)
```

Comment:
Can we keep this warning for local files? I'm not sure if there is a sensible case where you can read the file but not stat it, but if there is, I would like to inform the user that we are actually _not_ watching a file he expects us to watch...

Reply:
Yep! I've added a check, let me know if that is what you had in mind.